templates/inspec/examples/google_pubsub_topics.erb

```diff
@@ -3,13 +3,13 @@
 describe google_pubsub_topics(project: <%= gcp_project_id -%>) do
   it { should exist }
   its('names') { should include <%= doc_generation ? "'#{topic['name']}'" : "topic['name']" -%> }
-  its('count') { should eq 2 }
+  its('count') { should be >=1 }
 end
 
 describe.one do
   google_pubsub_topics(project: <%= gcp_project_id -%>).names.each do |topic_name|
     describe google_pubsub_topic(project: <%= gcp_project_id -%>, name: topic_name) do
-      its('name') { should eq <%= doc_generation ? "'#{topic['name']}'" : "topic['name']" -%> }
+      it { should exist }
```

Comment:
My standard thing I always say (so take it with a grain of salt) 

Could we auto-generate this block? This whole block looks the same across many singular resource. Especially with the `should exist` notation.